src/codegen/codegen_c_visitor.hpp

```diff
@@ -22,12 +22,11 @@
 #include <string>
 #include <utility>
 
-#include <fmt/format.h>
-
 #include "codegen/codegen_info.hpp"
 #include "codegen/codegen_naming.hpp"
 #include "printer/code_printer.hpp"
 #include "symtab/symbol_table.hpp"
+#include "utils/logger.hpp"
```

Comment:
`utils/logger.hpp` does not seem required in this header.

Reply:
@tristan0x : that's true! As mentioned in the commit message, `spdlog ships it's internal fmt library` and there are some issues/conflicts with using `spdlog` together `fmt` (which is required for `fmt::literals`).

Omar / Mike tried to fix that in #273 and #279 but still had issues (I am not intimately familiar). In this PR, I am removing use of `fmt` include and only keeping spdlog as it provides fmt as well. (Currently master is bit broken because of this)

Reply:
I mentioned existence of `SPDLOG_FMT_EXTERNAL`, which would probably clear a lot of this up in: https://github.com/BlueBrain/nmodl/pull/265#issuecomment-595669150

Hopefully it works, thus allowing the same version of `FMT` across the whole project.

Reply:
I was trying this option yesterday quickly and I got other compilation errors. (but I should spend bit more time to understand the root cause...)

Reply:
I will address external fmt usage separately in https://github.com/BlueBrain/nmodl/issues/332